packages/gatsby/src/bootstrap/load-plugins/__tests__/validate.js

```diff
@@ -15,7 +15,7 @@ const {
   handleMultipleReplaceRenderers,
   warnOnIncompatiblePeerDependency,
 } = require(`../validate`)
-const getLatestAPIs = require(`../../../utils/get-latest-apis`)
+const { getLatestAPIs } = require(`../../../utils/get-latest-apis`)
```

Comment:
```suggestion
import { getLatestAPIs } from "../../../utils/get-latest-apis"
```

Reply:
not necassary as these files are still js. When they converted they will switch to import